src/methods/modular_programs/easy/q6/Century.java

```diff
@@ -0,0 +1,31 @@
+package methods.modular_programs.easy.q6;
+
+/*
+Write a Java Program that asks users for the year that they were born in, then returns what century they were born in.
```

Comment:
Change question text to:
`
Write a Java Program that asks users for the year they were born, then returns what century that year is in. The program should have a method called centuryCalc to determine this, which works for any century from the 1500's to 2000's. If the input is in any other century, return "unknown century" .
`